includes/course-theme/class-sensei-course-theme-option.php

```diff
@@ -155,9 +154,6 @@ public static function should_use_learning_mode() {
 	 * @return boolean
 	 */
 	public static function should_override_theme() {
```

Comment:
Not sure if this is the best approach, I am thinking since we are removing the setting then we should return false here so theme styles are not overriden.